.circleci/config.yml

```diff
@@ -381,6 +381,7 @@ jobs:
       - checkout
       - run: make sync
       - run: make lint
+      - run: make yamllint
```

Comment:
oh actually didn't notice this was just added to the lint job not a new job, so nevermind.

It still could just be in `make lint` though :slightly_smiling_face: 

Looks like the image doesn't have yamllint though: `bin/yaml_lint.sh: line 33: yamllint: command not found`